src/WallpaperEngine/Core/CVideo.h

```diff
@@ -29,14 +27,13 @@ namespace WallpaperEngine::Core::Objects
         void getNextFrame ();
         void writeFrameToImage (irr::video::IImage* image);
```

Comment:
The functions to read the video from the file and creating the texture should be on the Render side instead of the Core side.